guiEditor/imgs/downArrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" width="17" height="17">

```

Comment:
Should this file be shared? or is it GUI-Editor only? Feels like a very generic svg file